src/engine/render/opengl_renderer/OpenGLRenderer.cpp

```diff
@@ -1,5 +1,5 @@
-#include "OpenGLRenderer.hpp"
-#include "gl_core_3_2.h"
+#include "engine/render/opengl_renderer/OpenGLRenderer.hpp"
```

Comment:
>Why do you changed to full path, it is located in the same directory ?
To keep code style the same across all sources, this is a good practice

Reply:
Yes, but please account specific of my tast, for me clearer use #include "file" if it is in the same directory

Reply:
>account specific of my tast
?

Reply:
I insist not because 'i just want to', but because full path has few benefits:
1) Other programmers who'll investigate your code will clearly see where file is located: #include "foo.hp" (not clear where to find this file), #include "engine/systems/subsystems/foo.hpp" (instantly see where to find this file
2) Other peoples don't know about your tastes, so, having common and clear standards in code base is a good approach for open source project

Reply:
If you don't want to use really full path, i propose to use full path relative to current module. 
Not #include "engine/render/opengl_renderer/OpenGLRenderer.hpp" but #include "render/opengl_renderer/OpenGLRenderer.hpp"